### PR TITLE
Updated .testing.pylintrc to match newer versions of pylint

### PR DESCRIPTION
**Pull request by justinta**
### What does this PR do?

Updates .testing.pylintrc 
### What issues does this PR fix or reference?

None
### Previous Behavior

pylint would throw lots of lint errors
### New Behavior

disables some of the new errors that are not relevant right now
### Tests written?
- [ ] Yes
- [x] No

### CODE REVIEW

File: salt/beacons/btmp.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def beacon(config):
             __context__[LOC_KEY] = fp_.tell()
             pack = struct.unpack(FMT, raw)
             event = {}
-            for ind in range(len(FIELDS)):
+            for ind in enumerate(FIELDS):
```
Comment (s0undt3ch) [inline salt/beacons/btmp.py]: Doesn't enumerate return `(idx, value)`?

Comment (jtand) [inline salt/beacons/btmp.py]: Oh, your right. it does. I forgot about that. What do you suggest here? leave it as range(len())?

Comment (s0undt3ch) [inline salt/beacons/btmp.py]:
```python
for idx, field in enumerate(FIELDS):
    event[field] = pack[ind]
    if isinstance(event[field], str):
        event[field] = event[field].strip('\x00')
```
?

Comment (jtand) [inline salt/beacons/btmp.py]: Could we just do
```python
for ind, fields in enumerate(FIELDS):
    event[FIELDS[ind]] = pack[ind]
    if isinstance(event[FIELDS[ind]], str):
        event[FIELDS[ind]] = event[FIELDS[ind]].strip('\x00')
```

Basically nothing changes except add the extra variable for unpacking. I could be very wrong in my thinking here, so feel free to correct me :)

Comment (s0undt3ch) [inline salt/beacons/btmp.py]: Pylint will complain of the unused `fields`(from your code).

Comment (jtand) [inline salt/beacons/btmp.py]: Ah, right. Ok, cool. I'll update.

**Comment by jtand** [comment]
@s0undt3ch How does that look?

File: salt/beacons/sh.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def beacon(config):
                      'tag': pid}
             if 'execve' in line:
                 comps = line.split('execve')[1].split('"')
-                for ind in range(len(comps)):
+                for ind in enumerate(comps):
```
Comment (s0undt3ch) [inline salt/beacons/sh.py]: Missed this one :)

**Comment by jtand** [comment]
@s0undt3ch Ok, that should do it :)

**Comment by s0undt3ch** [comment]
Yep, this looks good.

**Comment by jtand** [comment]
Go Go Jenkins!

**Pull request merged**